languages/ruby/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,5 @@
+# After
+
+- The [rubymonstas string interpolation tutorial][ruby-for-beginners.rubymonstas.org-interpolation] digs deeper into string interpolation.
```

Comment:
```suggestion
The key thing to remember about Ruby strings is that they are objects that you call methods on. You can find all the methods in the [Ruby docs](...)."

It's also worth knowing that strings can be created using single quotes (`'`) or double quotes (`"`), the only different being that you can [interpolate](by-for-beginners.rubymonstas.org-interpolation) inside double quotes. You can also create strings using the [block syntax](...) or using the `%q` and `%Q` helpers.
```

I think we can make this even better. I'll leave the choice of links etc up to you. I'm not blocking on this so if you want to it as an extra PR that'd be fine, but it might be nice to get this PR "perfect" as an examplar.

Reply:
I'm happy to keep working on it until it's "perfect" 😄 I added one more commit with your suggestions and a couple more things, let me know what you think